### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Palakfs**

### CODE REVIEW

**Comment by beingnoble03** [comment]
Please resolve the merge conflicts, remembering the steps I took to resolve conflicts previously in the lecture :)

File: README.md [hunk 1]
```diff
@@ -2,4 +2,8 @@ Add you name and github profile here. Have fun with MERGE CONFLICTS (if you were
 
 - [Noble Mittal](https://github.com/beingnoble03)
 - [Amrit Prakash](https://github.com/solo-daemon)
+
```
Comment (beingnoble03) [inline README.md:5]: remove this empty line.

File: README.md [hunk 2]
```diff
@@ -2,4 +2,8 @@ Add you name and github profile here. Have fun with MERGE CONFLICTS (if you were
...
+- [Palak Gupta](https://github.com/Palakfs)
+
```
Comment (beingnoble03) [inline README.md:7]: same

File: README.md [hunk 3]
```diff
@@ -2,4 +2,8 @@ Add you name and github profile here. Have fun with MERGE CONFLICTS (if you were
...
 - [Aditya Karwa](https://github.com/AdityaK-github)
+
```
Comment (beingnoble03) [inline README.md:9]: same

**Review by beingnoble03 — CHANGES_REQUESTED** [review]

**Comment by Palakfs** [comment]
Please review now.

**Pull request merged**